angular-workspace/projects/example-client-app/src/app/app.module.ts

```diff
@@ -64,11 +67,13 @@ import { HeaderComponent } from './header/header.component';
         NimbleAnchorTreeItemModule,
         NimbleTableModule,
         NimbleTableColumnTextModule,
+        NimbleTableColumnAnchorModule,
         NimbleBannerModule,
         RouterModule.forRoot(
             [
                 { path: '', redirectTo: '/customapp', pathMatch: 'full' },
-                { path: 'customapp', component: CustomAppComponent, title: 'Angular All Components Demo - Nimble Design System - NI' }
+                { path: 'customapp', component: CustomAppComponent, title: 'Angular All Components Demo - Nimble Design System - NI' },
+                { path: 'customapp2', component: CustomApp2Component, title: 'Angular All Components Demo - Nimble Design System - NI' }
```

Comment:
Don't add additional routes to the example-client-apps

Reply:
I wanted to exercise the router integration, but if we're prioritizing parity above that, I can remove the second component, route, etc.